forwards_paper.tex

```diff
@@ -625,17 +593,14 @@ \subsection*{Recording the pedigree in forwards time}
 \end{figure}
 
 
-\krt{The following paragraph has a sentence mangling.  I think some comment is warrented about when it is useful to
-record selected mutations, like for ancient samples?}
```

Comment:
I've got something, but not about ancient samples.

Reply:
Looks good.